src/help/which.js

```diff
@@ -0,0 +1,9 @@
+module.exports = `
+Usage: which COMMAND
+Look for COMMAND in $PATH.
+
+      --help                   display this help and exit
+
+Report which bugs to <https://github.com/khuongduybui/cash>
```

Comment:
Should probably leave this upstream at `https://github.com/dthree/cash` for consistency and one source of all bugs for the project.